src/components/ColorPicker/ColorPickerItem/index.tsx

```diff
@@ -0,0 +1 @@
+export { ColorPickerItem, ColorPickerItemProps } from './ColorPickerItem';
```

Comment:
I think this will give us a warning.
If it does, it can be changed to ```export * from './ColorPickerItem';```

Reply:
done